packages/PlanLimitsUI/src/pages/Limits/WaterQuantity.tsx

```diff
@@ -0,0 +1,5 @@
+const format = Intl.NumberFormat();
+
+export default function waterQuantity(quantity: number, unit: string) {
```

Comment:
Only a nit, but could we name this (file and fn) according to what it does? E.g. `formatWaterQuantity`?

Also, since we're not exporting a component, should the file be pascalCased?

Reply:
Good nit, it started off as a Component and then changed.